ros/src/waypoint_updater/waypnt_updater.py

```diff
@@ -1045,32 +1083,41 @@ def get_max_accel(self, ptr_id):
             acc_diff = -0.1
         else:
             acc_diff = 0.1
+        if max_jerk == self.max_desired_jerk:
```

Comment:
Is there a reason not to do a difference check like in `get_min_stopping_distance()`?